docs/CHANGELOG.asciidoc

```diff
@@ -65,6 +66,317 @@ Review important information about the {kib} 8.x releases.
+* Adds a link to Discover to view good/bad events in the event panel ({kibana-pull}178008[#178008]).
+* Adds customization for Virtual Columns in Field List ({kibana-pull}177626[#177626]).
+* Adds dependencies for Burn Rate rule suppression ({kibana-pull}177078[#177078]).
+* Adds grouping by multiple values when creating SLOs, allowing for dynamic creation of multiple SLOs from a single SLI definition ({kibana-pull}175063[#175063]).
+Uptime::
+* Adds Monitor public API ({kibana-pull}169928[#169928]).
+
+For more information about the features introduced in 8.14.0, refer to <<whats-new,What's new in 8.14>>.
+
+[[enhancements-and-bug-fixes-v8.14.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.14.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.14.0]]
+=== Enhancements
+Alerting::
+* Adds history chart for multiple conditions ({kibana-pull}180578[#180578]).
+* Show Alerting rule JSON for API requests ({kibana-pull}180085[#180085]).
+* Implement tabbed design for existing alert detail pages ({kibana-pull}179529[#179529]).
+* Adds new rule type selection modal ({kibana-pull}179285[#179285]).
+* Moves alerts filter controls to `@kbn/alerts-ui-shared` package ({kibana-pull}179243[#179243]).
+* Improves alerts table actions column performance ({kibana-pull}178632[#178632]).
+* Adds Insights component to alerts details ({kibana-pull}178330[#178330]).
+* Adds support to dhow the number of additional filters that are applied on the alerts table ({kibana-pull}177275[#177275]).
+* Adds error boundary to AlertsTable ({kibana-pull}176412[#176412]).
+* Improves the performance of `join_by_key` ({kibana-pull}175177[#175177]).
+APM::
+* Show Universal Profiling data on transaction details page ({kibana-pull}176922[#176922]).
+Connectors::
+* Adds support for the Jira connector API to support the `otherFields` property to pass additional fields to be used when updating or creating issues via the link:https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-post[Jira API] ({kibana-pull}178627[#178627]).
+Dashboard::
+* Reorganized panel actions in the actions menu ({kibana-pull}178596[#178596]).
+* Adds panel styling improvements ({kibana-pull}178139[#178139]).
+* Adds "Apply" button to stop controls selections being automatically applied ({kibana-pull}174714[#174714]).
+Elastic Security::
+For the Elastic Security 8.14.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+ES|QL::
+* Implicit casting changes ({kibana-pull}182989[#182989]).
+* Adds validation and auto-complete for `date_diff` ({kibana-pull}182513[#182513]).
+* Adds more functions to the validator ({kibana-pull}180640[#180640]).
+* Adds improvements to the AI assistants handling where a generated {esql} query has syntax errors ({kibana-pull}179919[#179919]).
+* Adds a chart switcher to {esql} in *Lens* ({kibana-pull}177790[#177790]).
```

Comment:
Let's add more details here: 

`Adds an enhanced chart switcher to *Lens* inline editing flyout`